proto/dclupgrade/query.proto

```diff
@@ -21,6 +22,16 @@ service Query {
 		option (google.api.http).get = "/dcl/dclupgrade/proposed-upgrades";
 	}
 
+// Queries a ApprovedUpgrade by index.
+	rpc ApprovedUpgrade(QueryGetApprovedUpgradeRequest) returns (QueryGetApprovedUpgradeResponse) {
+		option (google.api.http).get = "/dcl/dclupgrade/approved_upgrade/{name}";
```

Comment:
Please correct the endpoint. Plural nouns should be used for entities and dashes should be used as the words separator (see `ProposedUpgrade`).

Reply:
fixed

Reply:
fixed